.circleci/config.yml

```diff
@@ -51,3 +51,13 @@ jobs:
               chown -R 65534:65534 doc
               ./run.sh update
             fi
+      - run:
+          name: Run the sanity tests toward the real deployed cluster
```

Comment:
I guess all clusters are deployed. Maybe rephrase that this is towards the `prod` cluster i.e. `http://urbanization.qlikcore.com/`?